osm2gtfs.py

```diff
@@ -298,5 +293,34 @@ def load_config(file):
 
     return config;
 
+def load_agency(config):
+    '''
+    Loads agency data from a json config file. The valid keys under the json
+    "agency" object correspond to the transitfeed.Agency._FIELD_NAMES.
+
+    return a transitfeed.Agency object
+'''
+
+    data_dict = {}
+    for field_name in transitfeed.Agency._FIELD_NAMES:
+        try:
+
+            field_value = config["agency"][field_name]
+            if field_value is not None and field_value != "":
+                data_dict[field_name] = field_value
+
+        except KeyError:
+            sys.stderr.write("key '"+field_name+"' was not found on the config file.")
```

Comment:
Sometime later, we should maybe distinguish here between required and optional fields or does the validation already print warnings for that?